tests/TestPscPublish.hs

```diff
@@ -73,3 +65,15 @@ testPackage dir = pushd dir $ do
         putStrLn ("psc-publish tests failed on " ++ dir ++ ":")
         putStrLn (show other)
         exitFailure
+
+-- | Fail with a helpful error message when preparePackage fails
+preparePackageError :: Publish.PackageError -> IO ()
+preparePackageError e = do
+  Publish.printError e
```

Comment:
I had that planned and then forgot about it. Currently it's `UserError (BowerJsonNotFound)` that gets thrown. Is it fragile to check for that?